### ISSUE TITLE
Lost on install - 3 hours into it

### ISSUE THREAD

**Issue by techswa** [lifegadget/ember-cli-bootstrap-sassy#41]
I keep getting an error message and then I will uninstall everything and try again after reading through SO and Google.  I'm not sure if I have a version issue or just bad luck.  I'm on OS X and have the following stack:
Ember-cli 2.6.2
Node v4.4.5

I have installed $ npm install --save-dev ember-cli-sass

> node-sass@3.8.0 

But when I try
$ npm install --save-dev ember-cli-sass

I get:
$ ember install ember-cli-bootstrap-sassy
`libsass` bindings not found. Try reinstalling `node-sass`?
Error: `libsass` bindings not found. Try reinstalling `node-sass`?
    at getBinding (/Users/swa/Documents/Dev/bookmarks/node_modules/ember-cli-bootstrap-sass/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/node_modules/node-sass/lib/index.js:22:11)
    at Object.<anonymous> (/Users/swa/Documents/Dev/bookmarks/node_modules/ember-cli-bootstrap-sass/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/node_modules/node-sass/lib/index.js:188:23)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/swa/Documents/Dev/bookmarks/node_modules/ember-cli-bootstrap-sass/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/index.js:5:12)
    at Module._compile (module.js:409:26)
If you have any direction for me to proceed I would be very grateful!

**Comment by yankeeinlondon** [comment]
a few things ...
- you make reference to `ember-cli-sass`; there is/was an addon with that name quite a long time back which -- if I remember right -- is why I named it _sassy_. I think in this case this was just a typo so I'll precede with that assumption
- you installed the addon with `npm install` and while that installs the addon it doesn't execute the addon's install generator. Instead it's a better idea to install addons with `ember install` (I make this same mistake quite often)
- In this case, I believe this is indeed the problem you're running into as this addon depends on `ember-cli-sass` which in turn depends on `lib-sass`. 

Let me know if this sorts out your problem. Good luck.

**Comment by techswa** [comment]
Thanks.  Its difficult not to feel dumb when your not familiar with all the build tools:)  I figured out how to "rip" out the wrong libraries and start over.  I now have :

bower.json {
...
    "bootstrap": "~3.3.5",
    "bootstrap-sass": "^3.3.6"
}

package.json {
....
    "ember-cli-bootstrap-sassy": "0.5.3",
}

I have also created the app.scss with the contents of:

@import "bootstrap";
@import "bootstrap/theme";

I am no longer receiving error messages, but styling isn't being applied.  My guinea pig code is as follows:
``

```
<div class='container'>

<h3>List of Bookmarks</h3>

<table style='table'>
  <thead>
    <tr>
      <th>Id</th><th>Link</th><th>Category</th><th>Project</th><th>Action</th>
    </tr>
  </thead>
  <tbody>
  {{#each model as |bookmark|}}

  <tr>
    <td>{{bookmark.id}}</td><td><a href='http://{{bookmark.url}}' target='_blank'>{{bookmark.name}}</a></td><td>{{bookmark.description}}</td><td>{{bookmark.category}}</td><td>{{bookmark.project}}</td><td>{{bookmark.action}}</td>
  </tr>
  {{/each}}
</tbody>
</table>
</div>

```

It displays(html) correctly, just no styles. Am I missing something else, comma, period, you know, the normal crap?

Thank you for creating this code. I hope my dialog with you will save others from the same fate.

**Comment by yankeeinlondon** [comment]
A few more comments:
- you do not need/want to have `bootstrap` in your **bower.json**, just `bootstrap-sass`
- does making changes to your `styles/app.scss` make any changes? Try some basic styling adjustments. My guess is that you're still not using "ember-cli-sass". Make sure you install this with `ember install ember-cli-sass".

**Comment by techswa** [comment]
I ran bower uninstall bootstrap.  The command line showed that the library was removed, but the tree still showed the bootstrap branch and the bower.json file still had the entry.  I manually removed the entry and rebooted the ember server.  Still the same. I ran the 'ember install ember-cli-sass. Everything came back green, exiting with 'Binary is fine; exiting.'  I have added .text-center to the`<h3>` tag and it is centering.  

Your suggestions worked!

All is looking good now.  Note:  I did have the table using style instead of the class tag.  I was working on some Mandrill email templates that used inline styles just before this.  Difficult to switch between scripting languages sometimes.

Thanks for your support and thanks again for the package!

For the benefit of future troubleshooters
My current bower.json is below:

```

{
  "name": "bookmarks",
  "dependencies": {
    "ember": "~2.6.0",
    "ember-cli-shims": "0.1.1",
    "ember-cli-test-loader": "0.2.2",
    "ember-qunit-notifications": "0.1.0",
    "bootstrap-sass": "^3.3.6"
  }
}

```
